scripts/netContain/Dockerfile

```diff
@@ -22,7 +22,7 @@ FROM ubuntu:16.04
 # ENV export https_proxy=https://proxy.localhost.com:8080
 
 RUN apt-get update \
- && apt-get install -y openvswitch-switch=2.5.0-0ubuntu1 \
+ && apt-get install -y openvswitch-switch=2.5* \
```

Comment:
Should this be `2.5.*`?  I don't think a `2.55.x` release will come out any time soon, but still :joy: